pull-requests-allowed-labels-list

```diff
@@ -0,0 +1 @@
+new,changed,fixed, removed
```

Comment:
Maybe move it to .github folder?
And also filename, maybe `.pull-requests-allowed-labels-list` ?